src/custom/utils/operator/index.ts

```diff
@@ -191,31 +188,54 @@ export async function getPriceQuote(params: PriceQuoteParams): Promise<PriceInfo
   const [checkedBaseToken, checkedQuoteToken] = [checkIfEther(baseToken, chainId), checkIfEther(quoteToken, chainId)]
   console.log('[util:operator] Get Price from API', params)
 
-  return _getJson(
-    chainId,
-    `/markets/${toApiAddress(checkedBaseToken, chainId)}-${toApiAddress(checkedQuoteToken, chainId)}/${kind}/${amount}`,
-    'Quote'
-  )
+  try {
+    const response = await _fetchGet(
+      chainId,
+      `/markets/${toApiAddress(checkedBaseToken, chainId)}-${toApiAddress(
+        checkedQuoteToken,
+        chainId
+      )}/${kind}/${amount}`
+    )
+
+    return _handleQuoteResponse(response)
+  } catch (error) {
+    throw new QuoteError(UNHANDLED_QUOTE_ERROR)
+  }
 }
 
 export async function getFeeQuote(params: FeeQuoteParams): Promise<FeeInformation> {
   const { sellToken, buyToken, amount, kind, chainId } = params
   const [checkedSellAddress, checkedBuyAddress] = [checkIfEther(sellToken, chainId), checkIfEther(buyToken, chainId)]
   console.log('[util:operator] Get fee from API', params)
 
-  return _getJson(
-    chainId,
-    `/fee?sellToken=${toApiAddress(checkedSellAddress, chainId)}&buyToken=${toApiAddress(
-      checkedBuyAddress,
-      chainId
-    )}&amount=${amount}&kind=${kind}`,
-    'Quote'
-  )
+  try {
+    const response = await _fetchGet(
+      chainId,
+      `/fee?sellToken=${toApiAddress(checkedSellAddress, chainId)}&buyToken=${toApiAddress(
+        checkedBuyAddress,
+        chainId
+      )}&amount=${amount}&kind=${kind}`
+    )
+    return _handleQuoteResponse(response)
+  } catch (error) {
+    throw new QuoteError(UNHANDLED_QUOTE_ERROR)
+  }
 }
 
 export async function getOrder(chainId: ChainId, orderId: string): Promise<OrderMetaData | null> {
   console.log('[util:operator] Get order for ', chainId, orderId)
-  return _getJson(chainId, `/orders/${orderId}`, 'Order')
+  try {
+    const response = await _fetchGet(chainId, `/orders/${orderId}`)
+
+    if (!response.ok) {
+      const errorResponse: ApiErrorObject = await response.json()
+      throw new OperatorError(errorResponse)
```

Comment:
No mapping for Orders?

This is fine by me, just thinking "what would @anxolin say"? 

;)